### PR TITLE
Fixed broken link to writing benchmarks guide

### PR DESCRIPTION
**Pull request by a2xchip**

### CODE REVIEW

File: docs/separate-benchmark-source-set.md [hunk 1]
```diff
@@ -107,6 +107,6 @@ Set up a separate benchmark source set by following these simple steps:
 4. **Add Benchmarks**
 
    Place your benchmark code into the benchmark source set.
-   Refer to our [writing benchmarks guide](docs/writing-benchmarks.md) for a comprehensive guide on writing benchmarks.
+   Refer to our [writing benchmarks guide](/docs/writing-benchmarks.md) for a comprehensive guide on writing benchmarks.
```
Comment (qurbonzoda) [inline docs/separate-benchmark-source-set.md:110]: Given that in other guide documents we reference the relative location, I would suggest:
 ```suggestion
    Refer to our [writing benchmarks guide](writing-benchmarks.md) for a comprehensive guide on writing benchmarks.
```

Comment (qurbonzoda) [reply docs/separate-benchmark-source-set.md:110]: Could you please fix the same problem in line number 57?

**Review by qurbonzoda — CHANGES_REQUESTED** [review]

File: docs/separate-benchmark-source-set.md (hunk 1 above)
Comment (a2xchip) [reply docs/separate-benchmark-source-set.md:110]: @qurbonzoda Absolutely

Comment (a2xchip) [reply docs/separate-benchmark-source-set.md:110]: @qurbonzoda Done!

Comment (qurbonzoda) [reply docs/separate-benchmark-source-set.md:110]: https://github.com/Kotlin/kotlinx-benchmark/pull/192#discussion_r1526261349

What do you think about this suggestion?

Comment (a2xchip) [reply docs/separate-benchmark-source-set.md:110]: @qurbonzoda Hey, sorry. Didn't fully understood the suggestion. Let me check out overall convention used in the repository for doc links. For sure the most important thing is to keep it all consistent.

Comment (a2xchip) [reply docs/separate-benchmark-source-set.md:110]: Totally agree. Relative path is better for doc consistency. Updating.

**Comment by qurbonzoda** [comment]
Thanks!

**Comment by a2xchip** [comment]
@qurbonzoda Thank you for review

**Pull request merged**